vendor/windows_game_input/windows_game_input.odin

```diff
@@ -0,0 +1,1129 @@
+#+build windows
```

Comment:
Add this file to `examples/all` to that the CI can check it. There are a few coding style issues which have not been caught that I can see.

Reply:
Also the package location should probably be `vendor:windows/GameInput`